ADAL/src/request/ADAuthenticationRequest+AcquireToken.m

```diff
@@ -183,11 +191,49 @@ - (BOOL)checkExtraQueryParameters
     return url!=nil;
 }
 
+- (ADAuthenticationError *)checkClaims
+{
```

Comment:
I would agree, especially with the method name, it only makes sense to return a bool.